pandas/tests/indexes/interval/test_interval.py

```diff
@@ -857,6 +858,37 @@ def test_is_all_dates(self):
         year_2017_index = pd.IntervalIndex([year_2017])
         assert not year_2017_index.is_all_dates
 
+    def test_get_loc_errors(self):
```

Comment:
this should go in tests.indexes.interval.test_indexing.TestGetLoc

Reply:
@jbrockmendel thanks for the review, moved the tests.